src/common/connect.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { ConnectedHost } from './ConnectedHost';
+
+export function connect<T>(
+  component: React.ReactType,
+  storesToSubscribe: string[],
+  getProps: (props: T, ...stores) => T
```

Comment:
Not sure how you could possibly enforce that random context objects are of type anything. They are evaluated at runtime.